### ISSUE TITLE
Day of week constants

### ISSUE THREAD

**Issue by DanCardin** [python-pendulum/pendulum#819]
- [x] I am on the [latest](https://github.com/sdispater/pendulum/releases/latest) Pendulum version.
- [x] I have searched the [issues](https://github.com/sdispater/pendulum/issues) of this repo and believe that this is not a duplicate.

In pendulum version 2.1.2 the day of week constants are defined as:

```python
SUNDAY = 0
MONDAY = 1
TUESDAY = 2
WEDNESDAY = 3
THURSDAY = 4
FRIDAY = 5
SATURDAY = 6
```

In version 3 they're defined as:

```python
class WeekDay(IntEnum):
    MONDAY = 0
    TUESDAY = 1
    WEDNESDAY = 2
    THURSDAY = 3
    FRIDAY = 4
    SATURDAY = 5
    SUNDAY = 6

MONDAY = WeekDay.MONDAY
TUESDAY = WeekDay.TUESDAY
WEDNESDAY = WeekDay.WEDNESDAY
THURSDAY = WeekDay.THURSDAY
FRIDAY = WeekDay.FRIDAY
SATURDAY = WeekDay.SATURDAY
SUNDAY = WeekDay.SUNDAY
```

Further, this translates into object-API level differences:
2.1.2
```python
>>> imp.today().weekday()
0
>>> pendulum.today().day_of_week
1
```

vs 3.0.0

```python
>>> pendulum.today().weekday()
0
>>> pendulum.today().day_of_week
<WeekDay.MONDAY: 0>
```

---

At the very least, this appears to be an undocumented breaking change, if it's otherwise an intended change in API.

but it would be nice for there to be a baked-in part of the API to obtain the original 0=sunday day-of-week integer mapping. That somewhat relates to https://github.com/sdispater/pendulum/issues/63. Except I'm more just looking to retain the original pendulum integer mapping that appears to have changed.

**Comment by sshah90** [comment]
Agree! We saw this issue with the new version and can't seem to find any documentation related to it.

**Comment by ashb** [comment]
I'm afraid it is now too late to change this back, if we wanted to change it back to 0 = Sunday that would require a v4 as it would now be a breaking change on v3.

I'm going to close this issue, but a PR to update the docs to highlight this would be welcomed.

**Comment by georgpfolz** [comment]
What can we expect in v4? Will it be changed back or will the change in v3 be kept?
I guess the change will be kept as it aligns with the iso standard and Python's datetime.
